lib/i18n/strings_it.i18n.json

```diff
@@ -0,0 +1,85 @@
+    "opacityLabel": "Opacità",
+    "fontLabel": "Font",
+    "fontValidator": "Seleziona un font.",
+    "fontSizeLabel": "Dimensione del font",
+    "textGapLabel": "Spaziatura del testo",
+    "textGapDescription": "La spaziatura tra il testo in ogni riga di testo",
+    "rowGapLabel": "Interlinea",
+    "rowGapDescription": "La spaziatura tra ogni riga di testo",
+    "preview": "Anteprima",
+    "save": "Salva",
+    "savedSuccess": "Salvataggio riuscito",
+    "savedFailure": "Impossibile salvare l'immagine(i)",
+    "settings": "Impostazioni",
+    "languages": "Lingue"
+  },
+  "buttons": {
+    "agree": "Accetto",
+    "cancel": "Annulla",
+    "ignore": "Ignora",
+    "turnOn": "Accendi"
+  },
+  "dialogs": {
+    "exitConfirm": {
+      "exit": "Uscita",
+      "cancel": "Annulla",
+      "title": "Chiudi conferma",
+      "description": "Sei sicuro di voler chiudere questa finestra?"
+    },
+    "permissions": {
+      "photos": {
+        "title": "Consenti l'accesso al tuo album",
+        "description": "Vai alle Impostazioni del tuo telefono per concedere a $appName l'autorizzazione a visitare il tuo album."
+      },
+      "storage": {
+        "title": "Consenti l'accesso al tuo archivio",
+        "description": "Accedi alle Impostazioni del tuo telefono per concedere a $appName l'autorizzazione ad accedere al tuo archivio."
+      }
+    },
+    "licenseDialog": {
+      "licenseDialogTitle": "Termini e Condizioni",
+      "licenseDialogContentContent": "La protezione della privacy e delle informazioni personali degli utenti è un principio fondamentale di $appName.",
+      "licenseDialogContentTip": "Di seguito è riportato l'elenco delle autorizzazioni richieste da questa APP:",
+      "licenseDialogContentPrefix": "Prima di utilizzare i servizi di questa APP, ti preghiamo di leggere attentamente e accettare il ",
+      "licenseDialogContentUserAgreement": "Contratto d'uso",
+      "licenseDialogContentAnd": " e ",
+      "licenseDialogContentPrivacyAgreement": "L'accordo sulla privacy",
+      "licenseDialogContentSuffix": ", inizia a utilizzare i nostri servizi dopo aver accettato tutti i termini.",
+      "iosPermissions": [
+        "Per poter caricare le immagini, $appName deve avere accesso alle tue foto. (NSPhotoLibraryUsageDescription)"
+      ],
+      "androidPermissions": [
+        "Per poter utilizzare il servizio Sentry, $appName deve avere accesso a Internet. (android.permission.INTERNET)",
+        "Per poter caricare le immagini, $appName deve avere accesso al tuo spazio di archiviazione. (android.permission.READ_EXTERNAL_STORAGE)",
+        "Per poter salvare le immagini, $appName deve accedere al tuo spazio di archiviazione. (android.permission.WRITE_EXTERNAL_STORAGE)",
+        "Per poter caricare le immagini, $appName deve avere accesso alle tue foto. (android.permission.READ_MEDIA_IMAGES)"
+      ]
+    }
+  },
+  "locales(map)": {
+    "en": "English",
+    "it": "Italiano",
+    "zh": "中文"
+  },
```

Comment:
1. `lib/i18n/strings.i18n.json` and `lib/i18n/strings_zh.i18n.json` need to be consistent with here.

The code should be:

```diff
"locales(map)": {
    "en": "English",
+   "it": "Italiano",
    "zh": "中文"
},
```

2. `ios/Runner/Info.plist` and `ios/Runner/Info-Pro.plist`

The code should be:

```diff
<key>CFBundleLocalizations</key>
<array>
	<string>en</string>
+       <string>it</string>
	<string>zh</string>
</array>
```